languages/java-cpg/pom.xml

```diff
@@ -0,0 +1,87 @@
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>de.jplag</groupId>
+        <artifactId>languages</artifactId>
+        <version>${revision}</version>
+    </parent>
+
+    <artifactId>java-cpg</artifactId>
+
+    <properties>
+        <cpg.version>8.0.0</cpg.version>
+        <cpg.groupId>de.fraunhofer.aisec</cpg.groupId>
+        <kotlin.version>1.9.0</kotlin.version>
+    </properties>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>${cpg.groupId}</groupId>
+            <artifactId>cpg-core</artifactId>
+            <version>${cpg.version}</version>
+
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.logging.log4j</groupId>
+                    <artifactId>log4j-slf4j2-impl</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>${cpg.groupId}</groupId>
+            <artifactId>cpg-language-java</artifactId>
+            <version>${cpg.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>${cpg.groupId}</groupId>
+            <artifactId>cpg-analysis</artifactId>
+            <version>${cpg.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-stdlib-jdk8</artifactId>
+            <version>${kotlin.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-test</artifactId>
+            <version>${kotlin.version}</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.jetbrains.kotlin</groupId>
+                <artifactId>kotlin-maven-plugin</artifactId>
+                <version>${kotlin.version}</version>
+                <configuration>
+                    <jvmTarget>1.8</jvmTarget>
+                    <args>
+                        <arg>-Xcontext-receivers</arg>
+                    </args>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>compile</id>
+                        <goals>
+                            <goal>compile</goal>
+                        </goals>
+                        <phase>process-sources</phase>
+                    </execution>
+                    <execution>
+                        <id>test-compile</id>
+                        <goals>
+                            <goal>test-compile</goal>
+                        </goals>
+                        <phase>test-compile</phase>
+                    </execution>
+                </executions>
+            </plugin>
```

Comment:
Why compile during process-sources?

```
             <executions>
                    <execution>
                        <id>compile</id>
                        <goals>
                            <goal>compile</goal>
                        </goals>
                        <phase>compile</phase>
                    </execution>
                </executions>
```

My typical configuration: https://github.com/dfuchss/MensaBot/blob/a019410cbe13b2bfa8b67b80f8b80e00623f3227/pom.xml#L154

Reply:
But maybe it's needed because of the mix of Java and Kotlin code ..

Reply:
This must have been auto-generated, I would not have been able to select the appropriate phase. 
If I set it to `compile`, then compilation fails with a `Symbol not found` message for the Kotlin classes. 
Thank you for the review!